en/ui-system/components/editor/richtext.md

```diff
@@ -40,18 +42,39 @@ Here is an example of the `size` and `color` tag:
-| i | Render text as italic font | `<i>This text will be rendered as italic</i>`| The tag name must be lowercase and cannot be written in `italic`. |
-| u | Add a underline to the text |`<u>This text will have a underline</u>`| The tag name must be lowercase and cannot be written in `underline`. |
-| on | Specify a event callback to a text node, when you click the node, the callback will be triggered. | `<on click="handler"> click me! </on>` | Except for `on` tag, `color` and `size` tags can also add `click` event parameter. eg. `<size=10 click="handler2">click me</size>` |
-| param | When the click event is triggered, the value can be obtained in the second attribute of the callback function. | `<on click="handler" param="test"> click me! </on>` | Depends on the click event. |
-| br | Insert a empty line | `<br/>` | `<br></br>` and `<br>` are both invalid tags. |
-| img | Add image and emoji support to your __RichText__. The emoji name should be a valid sprite frame name in the ImageAtlas. | `<img src='emoji1' click='handler' />` | Only `<img src='foo' click='bar' />` is a valid img tag. If you specify a large emoji image, it will scale the sprite height to the line height of the __RichText__ together with the sprite width. |
+| b       | Render text as bold font | `<b>This text will be rendered as bold</b>`| The tag name must be lowercase and cannot be written in `bold`. |
+| i       | Render text as italic font | `<i>This text will be rendered as italic</i>`| The tag name must be lowercase and cannot be written in `italic`. |
+| u       | Add a underline to the text |`<u>This text will have a underline</u>`| The tag name must be lowercase and cannot be written in `underline`. |
+| on      | Specify a event callback to a text node, when you click the node, the callback will be triggered. | `<on click="handler"> click me! </on>` | Except for `on` tag, `color` and `size` tags can also add `click` event parameter. eg. `<size=10 click="handler2">click me</size>` |
+| param   | When the click event is triggered, the value can be obtained in the second attribute of the callback function. | `<on click="handler" param="test"> click me! </on>` | Depends on the click event. |
+| br      | Insert a empty line | `<br/>` | `<br></br>` and `<br>` are both invalid tags. |
+| img     | Add image and emoji support to your __RichText__. The emoji name should be a valid sprite frame name in the ImageAtlas. | `<img src='emoji1' click='handler' />` | Only `<img src='foo' click='bar' />` is a valid img tag. If you specify a large emoji image, it will scale the sprite height to the line height of the __RichText__ together with the sprite width. |
+
+#### Optional attribute of img tag
+
+For better typography we have provided additional optional attributes for the img tag. You can use `width`, `height` to specify the size of the SpriteFrame. This will allow the image to be larger or smaller than the line height (but it will not change the line height).
```

Comment:
For better typography we have provided additional optional attributes for the img tag. You
-->
For better typography, additional optional attributes to the __img__ tag have been provided. Developers